Add meeting notes for December 2021 (#39)
dotnet-foundation/wg-maintainers: Meetings/2021-12-08-meeting-minutes.md

# Maintainers' Meeting 2021-21-08

## Meeting Attendees

- Rob Prouse - @rprouse
- Mattias Karlsson - @devlead
- Calvin A. Allen - @CalvinAllen
- Dan Siegel - @dansiegel
- James Newton-King - @JamesNK
- James Turner - @Turnerj
- Pascal Berger - @pascalberger
- Nils Andresen - @nils-a

## Recording

> **TBD**

Reviewer: @rprouse Not sure if this is a "good" place to discuss, but:
The recording for the last (second to last? Well, the "other" meeting..) are also still "TBD". Is that a "need the time to upload" issue, or rather a "don't know where to put it" issue? 
Is there anything someone can do to help?

Author: It is a find a don't know where to store it issue. I tried sharing from the foundation OneDrive but external sharing is blocked. I want to get that changed. We also discussed YouTube, but I'm not sure meeting recordings should be in the main Foundation YouTube feed.

Reviewer 2: Could do an unlisted video and just link it here? Wouldn't appear in anyone's subscription feeds then.

Author: I'll ask about that, thanks. I'm not going to hold up merging this though, so I'm going to resolve.

### Virtual Inc Staff Observers

- Nicole Miller

## Slides

## Agenda

- Review Action Items
- Discuss the process to leave the foundation

## Notes

### Process for Leaving the Foundation

- Reviewed the process to leave the Foundation as posted in the [GitHub discussion](https://github.com/dotnet-foundation/Home/discussions/68#discussioncomment-1774246).

> - After all comments have been reviewed, the board will meet with the lawyers to reverse the contracts that have been put in place, the board will work with the projects individually that have sent notices to leave.

Reviewer: This sentence sounds funny to me. (Could well be my English, though.)

> [...] the board will meet with the lawyers to reverse the contracts that have been put in place [...]

Sounds a bit like "all of the contracts". Maybe something like: 

```suggestion
- After all comments have been reviewed, the board will meet with the lawyers to reverse the contracts of the projects that have sent notices to leave. The board will work with each of the aforementioned projects individually.
```
